bitcoin/src/sign_message.rs

```diff
@@ -165,8 +164,8 @@ mod message_signing {
         use base64::prelude::{Engine as _, BASE64_STANDARD};
 
         use super::*;
-        use crate::prelude::String;
-
+        use crate::prelude::*;
+            
```

Comment:
In 16f75903ea99c88e5e1f8a792cd6c0342b191936:

Trailing whitespace.

Reply:
still!

Reply:
This PR has non-building patches in it, we aren't enforcing green CI on every patch yet are we?

Reply:
If we are I'll have to squash it into a single patch.

Reply:
We are not enforcing green CI on every patch, no, and I don't think it'll pass.

We are enforcing that *my* CI passes on every patch but I can poke at that to temporarily disable or pin nightly.